_includes/scenarios/integrate/data-model-overview.md

```diff
@@ -294,21 +294,86 @@ A Request allows you to ask another Identity to do something. What this "somethi
+
+| Name                  | Type                                                                                                                                                                                             | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| attribute             | [`IdentityAttribute`]({% link _docs_integrate/data-model-overview.md %}#identityattribute) \| [`RelationshipAttribute`]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute) | The IdentityAttribute or RelationshipAttribute to share. This is not the LocalAttribute but the content data structure of the Attribute. <br>The owner of the Attribute which should be shared can only be the sender Identity.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
I noticed that in the Data Model Overview the spelling "Local Attribute" is used, but also "LocalAttribute".

For reasons of consistency, we may have to decide which variant we want to use. IMHO, I would prefer "LocalAttribute" compared to "Local Attribute"